src/components/sequence-submission.jsx

```diff
@@ -0,0 +1,94 @@
+import React, {
+  useState,
+  useEffect,
+  useCallback,
+  useRef,
+  Fragment,
+} from 'react';
+import PropTypes from 'prop-types';
+import validateSequences from '../validators/sequenceValidator';
+import Message from './message';
+import '../styles/components/sequence-submission.scss';
+
+const SequenceSubmission = ({
+  value,
+  onChange,
+  placeholder,
+}) => {
+  const [error, setError] = useState('');
+  const ref = useRef();
+  const autoFocus = () => {
+    ref.current.focus();
```

Comment:
Not sure about that behaviour by default. I don't like it in general, and it's not really nice for accessibility reasons.
Is there any specific reason why this is here?

Reply:
No specific reason. I don't know what accessibility issues the auto-focus can cause, but either try to improve it or remove it altogether.

Reply:
It's basically the equivalent of being teleported in an unexpected area. A screenreader would jump directly there without any context of where they are within the page.
Also, it might not be the main input for some people as we'll have a different input to load a sequence from an accession.

Reply:
Okay, perfect, I didn't know that! The auto-focus is now removed!